editor/code_editor.cpp

```diff
@@ -467,7 +487,6 @@ bool FindReplaceBar::search_prev() {
 		}
 		col = text_editor->get_line(line).length();
 	}
-
```

Comment:
Unrelated (and IMO unnecessary) whitespace change, should be restored

Reply:
Aight, it's probably because I added and then removed code. Whitespaces tend to be arbitrary so this stuff happens.

Changed it back for my next push.